examples/smbpasswd.py

```diff
@@ -46,7 +46,9 @@
 
 class SMBPasswd:
 
-	def __init__(self, domain, username, oldPassword, newPassword, oldPwdHashLM, oldPwdHashNT, newPwdHashLM, newPwdHashNT, hostname):
+	def __init__(self, domain='', username='', oldPassword='', newPassword='',
+				oldPwdHashLM='', oldPwdHashNT='', newPwdHashLM='', newPwdHashNT='',
+				hostname='', doKerberos=False, kdcHost=None):
```

Comment:
Default values for parameters are not necessary, all of the parameters are passed to SMBPasswd at __main__, SMBPasswd is not used anywhere else.